### DISCUSSION TITLE
Possible to freeze panes?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#305] [Ideas]
Hello,

Love this csv plugin, but can it freeze panes in Grafana?  I have imported a rather large csv and it would be great to freeze the top row and first column, possible?

**Comment by zoltanbedi** [comment]
This would be a feature request for whatever visualization you use with this data source.

**Reply by G0nz0uk** [reply]
It's Grafana, so raise with them?

**Reply by zoltanbedi** [reply]
Sure you can.
